introduction.md

```diff
@@ -1,3 +1,8 @@
 My name is Mariano Rodríguez, I'm 22 years old
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?